sections/plate_notation.tex

```diff
@@ -1,53 +1,15 @@
-It is also possible to visualize the documents by, e.g., sorting by the highest topic proportions in $\theta_d$.
-Another possibility of exploration is finding similar documents by using a distribution distance function on the topic proportions $\theta_d$ between documents~\cite{blei2009topic}.
+\section{Our Topic Models}\label{sec:plate_notation}
+In this section we will detail the modifications that we have implemented to the existing models, outlined in \autoref{sec:preliminaries}.
+We will be presenting three models, one for each of the metadata fields detailed in \autoref{sec:dataset}.
+Two of the models: the Author and Category metadata models, are based on the Author-topic \gls{lda} explained in \autoref{subsec:auth_prelim}.
+Lastly the Taxonomy metadata model is based on \gls{pam}, which is explained in \autoref{subsec:pachinko_prelim}.
 
 \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
-We model both of the metadata fields 'Author' and 'Category' similarly to the model presented by \citet{author_topic_2012}.
-In this model, there are no document-topic distributions $\theta$.
-Instead, each author and category has its own topic distribution.
-This is based on the assumption that authors prefer to write about specific topics, and that categories of the articles were chosen based on the content of the finished article or that local newspapers have their own unique topic preferences.
-
-Our topic models are \glspl{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have an influence on both word distributions and other metadata.
+We model both of the metadata fields 'Author' and 'Category' similarly to the model by \citet{author_topic_2012} presented in \autoref{subsec:auth_prelim}.
+The category model builds on the assumption that categories of the articles were chosen based on the content of the finished article and that local newspapers have their own unique topic preferences.
+We find this model to be generally applicable to most metadata information assuming that the metadata is either chosen based on the text of the documents (as with the category metadata) or that it has some impact of the text of the document (as with the author metadata).
 
-\citet{author_topic_2012} describe the author-topic model where for each document $d$, they assign a vector of authors $a_d$ from a set of authors $A$, and for each word draw an author $x$ from this vector.
-However, for our category-topic model and our own author-topic model, each document $d$ is associated with one category $c_d$ from a set of categories $C$ and one author $a_d$ from a set of authors $A$, rather than a vector.
+For our category-topic model and our own author-topic model, each document $d$ is associated with only one category $c_d$ from a set of categories $C$ and only one author $a_d$ from a set of authors $A$, rather than a vector.
```

Comment:
```suggestion
For our category-topic model and our author-topic model, each document $d$ is associated with only one category $c_d$ from a set of categories $C$ and only one author $a_d$ from a set of authors $A$, rather than a vector.
```
måske ref til author-topic